wm/property.go

```diff
@@ -103,6 +104,34 @@ func windowMinSize(x *xgbutil.XUtil, win xproto.Window) (uint, uint) {
 	return 0, 0
 }
 
+func windowApplyIncrement(x *xgbutil.XUtil, win xproto.Window, width uint16, height uint16) (uint16, uint16) {
```

Comment:
This is not actually applying anything - it is returning a value so verbs like calculate, round or similar would be clearer.
Perhaps even windowSizeWithIncrement?

Reply:
updated to be windowSizeWithIncrement